### ISSUE TITLE
Error: SimpleError { err: "Failed to rename ....

### ISSUE THREAD

**Issue by monyarm** [alucryd/oxyromon#32]
I set my ROMDIRECTORY to /mnt/Games, imported a dat file, and then ran 

```
oxyromon import-roms ~/Downloads/Megaman\ Battle\ Network\ \(U\)\ \[\!\].zip             
Processing "/home/monyarm/Downloads/Megaman Battle Network (U) [!].zip"
Processing "Megaman Battle Network (U) [!].gba"
Matches "Mega Man Battle Network (USA).gba"
Renaming "Megaman Battle Network (U) [!].gba" to "Mega Man Battle Network (USA).gba"
Moving to "/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba.zip"
Error: SimpleError { err: "Failed to rename \"/home/monyarm/Downloads/Megaman Battle Network (U) [!].zip\" to \"/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba.zip\", could not rename `/home/monyarm/Downloads/Megaman Battle Network (U) [!].zip` to `/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba.zip`" }
```

Thinking that it might be because it's zipped, i then tried 
```
oxyromon import-roms Mega\ Man\ Battle\ Network\ \(USA\).gba                
Processing "Mega Man Battle Network (USA).gba"
Matches "Mega Man Battle Network (USA).gba"
Moving to "/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba"
Error: SimpleError { err: "Failed to rename \"/home/monyarm/Downloads/Mega Man Battle Network (USA).gba\" to \"/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba\", could not rename `/home/monyarm/Downloads/Mega Man Battle Network (USA).gba` to `/mnt/Games/Nintendo - Game Boy Advance/Mega Man Battle Network (USA).gba`" }
```

The /mnt/Games/Nintendo - Game Boy Advance/ folder has been successfully created, but it refuses to move it over.

`/` `/mnt/Games` and `/home/` are each different partitions (`/home` is actually a bind mount to `/mnt/HDD/home`)

I'm running gentoo linux, and ran `cargo build --release` in order to build oxyromon

**Comment by monyarm** [comment]
Download the prebuild release has the same issue

**Comment by alucryd** [comment]
Thanks for reporting the issue. It's the different filesystems apparently: https://docs.rs/async-std/latest/async_std/fs/fn.rename.html

In such cases I'll have no choice but to copy the file and then delete the source. Will do it over the weekend, please stay tuned.

**Comment by alucryd** [comment]
Should be fixed in e95b000330eacfafd65d5da6204cd1c28ea2bab3.

@monyarm Are you able to build this commit? I have a lot going on in the develop branch so I'm not comfortable with releasing 0.11 yet, however I can cherry-pick that in the 0.10 branch and produce a 0.10.2 hotfix release if you'd like.

**Comment by alucryd** [comment]
Just released 0.11.0, please let me know if you have run into any other issue.
